packages/markdown/components/Code/index.jsx

```diff
@@ -5,12 +5,32 @@ const PropTypes = require('prop-types');
 // apps. Necessary because of people like this:
 // https://github.com/codemirror/CodeMirror/issues/3701#issuecomment-164904534
 const syntaxHighlighter = typeof window !== 'undefined' ? require('@readme/syntax-highlighter') : false;
+const copy = require('copy-to-clipboard');
+
+const remapLang = {
```

Comment:
I really hate how we have this stuff spread across this package, `@readme/syntax-highlighter`, and in ReadMe. One of these days we should consolidate all of our code rendering into UI or something.

Reply:
Strong agree.